### PR TITLE
Make bootkube wait for ceo to finish before declaring success

### PR DESCRIPTION
**Pull request by retroflexer**

### CODE REVIEW

File: data/data/bootstrap/files/usr/local/bin/bootkube.sh.template [hunk 1]
```diff
@@ -367,6 +367,16 @@ bootkube_podman_run \
 	"${CLUSTER_BOOTSTRAP_IMAGE}" \
 	start --tear-down-early=false --asset-dir=/assets --required-pods="openshift-kube-apiserver/kube-apiserver,openshift-kube-scheduler/openshift-kube-scheduler,openshift-kube-controller-manager/kube-controller-manager,openshift-cluster-version/cluster-version-operator"
 
+if [ ! -z "$CLUSTER_ETCD_OPERATOR_IMAGE" ]
+then
+	echo "Waiting for ceo to finish..."
+	bootkube_podman_run \
+		--quiet \
```
Comment (alaypatel07) [inline data/data/bootstrap/files/usr/local/bin/bootkube.sh.template]: can remove this flag bootkube_podman_run has this?

Comment (retroflexer) [reply data/data/bootstrap/files/usr/local/bin/bootkube.sh.template]: No quiet?

Comment (retroflexer) [reply data/data/bootstrap/files/usr/local/bin/bootkube.sh.template]: What about volume?

File: data/data/bootstrap/files/usr/local/bin/bootkube.sh.template [hunk 2]
```diff
@@ -367,6 +367,15 @@ bootkube_podman_run \
 	"${CLUSTER_BOOTSTRAP_IMAGE}" \
 	start --tear-down-early=false --asset-dir=/assets --required-pods="openshift-kube-apiserver/kube-apiserver,openshift-kube-scheduler/openshift-kube-scheduler,openshift-kube-controller-manager/kube-controller-manager,openshift-cluster-version/cluster-version-operator"
 
+if [ ! -z "$CLUSTER_ETCD_OPERATOR_IMAGE" ]
+then
+	echo "Waiting for ceo to finish..."
+	bootkube_podman_run \
+		--volume /etc/kubernetes:/etc/kubernetes:z \
+		"${CLUSTER_ETCD_OPERATOR_IMAGE}" \
+		/usr/bin/wait-for-ceo --kubeconfig /etc/kubernetes/kubeconfig
```
Comment (alaypatel07) [inline data/data/bootstrap/files/usr/local/bin/bootkube.sh.template]: @retroflexer This should be `/usr/bin/cluster-etcd-operator wait-for-ceo`
https://github.com/hexfusion/installer/blob/811d527e1e25eda0c26024d92db60a698e7fcf0c/data/data/bootstrap/files/usr/local/bin/bootkube.sh.template#L123

File: data/data/bootstrap/files/usr/local/bin/bootkube.sh.template [hunk 3]
```diff
@@ -367,6 +367,15 @@ bootkube_podman_run \
...
 	start --tear-down-early=false --asset-dir=/assets --required-pods="openshift-kube-apiserver/kube-apiserver,openshift-kube-scheduler/openshift-kube-scheduler,openshift-kube-controller-manager/kube-controller-manager,openshift-cluster-version/cluster-version-operator"
 
+if [ ! -z "$CLUSTER_ETCD_OPERATOR_IMAGE" ]
+then
+	echo "Waiting for ceo to finish..."
+	bootkube_podman_run \
+		--volume /etc/kubernetes:/etc/kubernetes:z \
+		"${CLUSTER_ETCD_OPERATOR_IMAGE}" \
+		/usr/bin/cluster-etcd-operator wait-for-ceo --kubeconfig /etc/kubernetes/kubeconfig
+fi
```
Comment (hexfusion) [inline data/data/bootstrap/files/usr/local/bin/bootkube.sh.template:379]: General style nit.

$binary \\
  $command \\
  —flags ...

Example: https://github.com/hexfusion/installer/blob/581ff0f21f59c0dc3972410194231435f523da67/data/data/bootstrap/files/usr/local/bin/bootkube.sh.template#L64-L70

**Review by hexfusion — COMMENTED** [review]
/lgtm

**Pull request merged**